src/hotspot/share/cds/filemap.hpp

```diff
@@ -473,7 +473,6 @@ class FileMapInfo : public CHeapObj<mtInternal> {
 
   static void allocate_shared_path_table(TRAPS);
   static void copy_shared_path_table(ClassLoaderData* loader_data, TRAPS);
-  static void clone_shared_path_table(TRAPS);
```

Comment:
I think the `copy_shared_path_table` could be removed as well.

Reply:
I've removed it.